bplan/serializers.py

```diff
@@ -49,18 +59,24 @@ def get_bplan_imVerfahren_count(self, object):
         return bplan_count - bplan_festgesetzt_count - bplan_aul_count - bplan_bbg_count
 
     def get_bplan_aul_count(self, object):
+        afs_behoer = self._get_param('afs_behoer')
         date = timezone.now().date()
         qs = BPlan.objects.filter(bezirk=object).filter(festg=False)
         qs = qs.exclude(aul_anfang__isnull=True, aul_ende__isnull=True)
-        count = qs.filter(aul_anfang__lte=date, aul_ende__gte=date).count()
-        return count
+        qs = qs.filter(aul_anfang__lte=date, aul_ende__gte=date)
+        if afs_behoer:
```

Comment:
This if is in all three functions. Maybe you want to write a function instead that provides the filtered queryset, and call that from the three functions.